_posts/2024-11-06-#30239.md

```diff
@@ -10,18 +10,38 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- Bitcoin Core considers a transaction output to be [dust](https://bitcoin.stackexchange.com/questions/10986/what-is-meant-by-bitcoin-dust) when its value is lower than the cost of creating and spending it. These outputs may be useful in particular circumstances, e.g. L2s, when the parent transaction cannot have fees for technical or accounting reasons.
 
+- Recommend reading from earlier review clubs:
+  - [#30352: Add PayToAnchor(P2A), OP 1 <0x4e73> as standard output script for spending](https://bitcoincore.reviews/30352)
```

Comment:
In similar style to what was done for n30352, would be good to highlight how this PR relates to n30352, how it's different, ... To give some context to people jumping in.

Reply:
Also, in this case, I'd put the link to n30352 at the top of the notes, and make it required reading

Reply:
Added more initial content, but will likely return and tweak